src/main/drivers/pwm_output.c

```diff
@@ -31,12 +31,14 @@
 #define MULTISHOT_5US_PW    (MULTISHOT_TIMER_MHZ * 5)
 #define MULTISHOT_20US_MULT (MULTISHOT_TIMER_MHZ * 20 / 1000.0f)
 
-#define DSHOT_MAX_COMMAND 47
-
 static pwmWriteFuncPtr pwmWritePtr;
 static pwmOutputPort_t motors[MAX_SUPPORTED_MOTORS];
 static pwmCompleteWriteFuncPtr pwmCompleteWritePtr = NULL;
 
+#ifdef USE_DSHOT
+loadDmaBufferFuncPtr loadDmaBufferPtr;
```

Comment:
I thought about function pointer vs. switch, but then I decided for function pointer for future extensibility. I think there is potential for more experimentation with with higher update rates / more robust protocols here.